src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Controllers/PolicyController.cs

```diff
@@ -120,5 +123,47 @@ public async Task<ActionResult> GetRolesWithAccess(string org, string app)
 
             return Ok(PolicyHelper.GetRolesWithAccess(policy));
         }
+
+        /// <summary>
+        /// Gets a list of resource policies for the list of org/apps
+        /// </summary>
+        /// <param name="orgAppList">The list of org/apps</param>
+        /// <returns>A list resourcePolicyResponses</returns>
+        [AllowAnonymous]
+        [HttpPost("/authorization/api/v1/resourcepolicies")]
```

Comment:
```suggestion
        [HttpPost("/authorization/api/v1/policies/GetPolicies")]
```
This makes it more in line with the existing WritePolicy endpoint and the other RPC endpoints as described in https://app.swaggerhub.com/apis/Altinn/Rights-3.0/0.10.0